### PR TITLE
Add new parameter css-validate to cloner

### PR DESCRIPTION
**Pull request by viskey98**
This will create a new parameter `css-validate` to set whether css validation is required or not.
Fixes issue #113

### CODE REVIEW

File: clone.py [hunk 1]
```diff
@@ -165,8 +165,11 @@ def _make_filename(self, url):
                     data = str(soup).encode()
                 with open(os.path.join(self.target_path, hash_name), 'wb') as index_fh:
                     index_fh.write(data)
-                if content_type == 'text/css':
-                    css = cssutils.parseString(data)
+                if content_type == 'text/css':                       
+                    if self.css_validate == "False":
+                        css = cssutils.parseString(data, validate=False)
```
Comment (afeena) [inline clone.py]: Maybe it's better to write:
```cssutils.parseString(data, validate=self.validate)```
In this case you don't need this if/else statement

**Comment by viskey98** [comment]
@afeena Keeping your suggestion in mind.
As the argparse gets input as string (even if we use `type=bool`, any input will validate to True as it is direct string to bool conversion). 
According to me there are two solutions for that:
1. Create a custom function `str_to_bool()` (in a desired way) and in argparse use `type=str_to_bool` .
2. Or else we have to take input as a string in argparse (using `type=str, default="True"`). And have to use an if-else statement before passing the parameter to the Cloner class or after. 
Now, I have changed the code and implemented the first one, please suggest any changes if required :)

**Comment by afeena** [comment]
@viskey98 good work :)

**Comment by viskey98** [comment]
@afeena Thank you :)

**Pull request merged**